rpm/tarantool.spec

```diff
@@ -89,23 +89,24 @@ BuildRequires: libunwind-devel
 %endif
 
 # Set dependences for tests.
-# Since Python 2 was deprecated then new OS do not have repositories
-# with its packages. It is not good way to create all these packages
-# manually and store it as backported. Let's wait when Python 3 will
-# be used for testing.
-%if (0%{?fedora} < 33 && 0%{?rhel} < 9)
-# Do not install unused Python 3 packages which
-# is default since Fedora 31 and CentOS 8.
-%if (0%{?fedora} >= 31 || 0%{?rhel} >= 8)
-BuildRequires: python2 >= 2.7
-BuildRequires: python2-six >= 1.9.0
-BuildRequires: python2-gevent >= 1.0
-BuildRequires: python2-yaml >= 3.0.9
-%else
-BuildRequires: python >= 2.7
-BuildRequires: python-six >= 1.9.0
-BuildRequires: python-gevent >= 1.0
-BuildRequires: python-yaml >= 3.0.9
+%if (0%{?rhel} != 6)
+BuildRequires: python3
+BuildRequires: python3-devel
+BuildRequires: python3-pip
+%if (0%{?rhel} >= 8 || 0%{?fedora} >= 28)
+BuildRequires: python3-gevent
+BuildRequires: python3-six
+BuildRequires: python3-yaml
+%endif
+%if (0%{?rhel} == 7)
+BuildRequires: python3-six
+BuildRequires: python3-gevent
+BuildRequires: python3-pyyaml
+%endif
+%if (0%{?sle_version} >= 1500)
+BuildRequires: python3-gevent
+BuildRequires: python3-six
+BuildRequires: python3-PyYAML
```

Comment:
Not a big problem, but from what I [see](https://github.com/tarantool/tarantool/pull/5751#discussion_r575500179), the following should be sufficient:

```
%if (0%{?rhel} != 6)
BuildRequires: python3
BuildRequires: python3-six
BuildRequires: python3-gevent
%if (0%{?sle_version} >= 1500)
BuildRequires: python3-PyYAML
%else
BuildRequires: python3-pyyaml
%endif
%endif
```

python3-devel and python3-pip are not necessary.

Reply:
updated conditions in spec